CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ Unreleased
 * Add support for onBeforeUnload in firefox selenium testing
 * Adding git to terra-node-base to be able to reference git branches while testing.
 
+### Changed
+* Update the Firefox test browser to use the `selenium/node-firefox:3.141.59-titanium` docker image
```

Comment:
Please move this to "unreleased" section.

Reply:
Changed here : https://github.com/cerner/terra-toolkit/pull/356/commits/50ac3e447dfe3961f6e15fdd57bf106335fd8844